get_imports.py

```diff
@@ -173,11 +174,23 @@ def get_libs_for_project(project_name):
         if file.endswith(".py"):
 
             found_imports = findimports.find_imports(f"{project_dir}{file}")
-
             for cur_import in found_imports:
                 cur_lib = cur_import.name.split(".")[0]
                 if cur_lib in bundle_data or cur_lib in community_bundle_data:
                     found_libs.add(cur_lib)
+                if cur_import.name.endswith(".*"):
```

Comment:
Is this literally looking for `foo.*`? A comment or example would help explain what's going on here. Thanks.

Reply:
@dhalbert Yes, it is technically looking literally for something like `foo.*`. However the property that it's searching within has already been processed by `findimports` some. 

`findimports` will take an original import line from a code.py file such as:
```
from foo.bar import *
```

and in the data returned from `findimport` the `name` property will be a string in the form of:`foo.bar.*`

This code is searching `cur_import.name` for that trailing star.

Perhaps this restriction to wildcard imports may not be strictly necessary, I added it because I noticed this specifically with the Karel guide code I'm working on which uses the import in this form and I wanted to try to minimally impact any other generated images. But it may be worth going back and seeing if we would want this in cases that aren't wildcard imports as well.

Reply:
> it's looking for `.*`, not just `*`

It could look for just the `*` and still work correctly I believe. I don't think I had a specific reason for choosing `.*`. Maybe just subconsciously anticipating slicing off the last 2 characters which it does in the next line.

I could change it if it's better without the dot.

Reply:
I updated the comment to `# findimports returns import name in the form of "foo.bar.*"` in the latest commits. 

Thanks for taking a look.